docs/install/prerequisites.rst

```diff
@@ -170,8 +146,11 @@ instructions specific to your distribution to add the necessary repositories.
 
                     .. code-block:: shell
 
+                        SUSEConnect -p sle-module-desktop-applications/{{ os_version }}/x86_64
+                        SUSEConnect -p sle-module-development-tools/{{ os_version }}/x86_64
+                        SUSEConnect -p PackageHub/{{ os_version }}/x86_64
                         sudo zypper addrepo https://download.opensuse.org/repositories/devel:/languages:/perl/{{ os_version }}/devel:languages:perl.repo
-                        sudo zypper addrepo https://download.opensuse.org/repositories/Education/{{ os_version }}/ Education
+                        sudo zypper addrepo https://download.opensuse.org/repositories/Education/{{ os_version }}/Education.repo
```

Comment:
You don't need the ".repo" on the end.  What this command does is it adds the .repo file to /etc/zypp/repos.d.  With the .repo at the end, it will add another .repo to the file since zypper automatically adds the .repo anyway.  So, you actually get "Education.repo.repo", running this command.  That being say, it will still work correctly anyway.

Reply:
I sync'd with @dbieleck and we confirmed the ".repo.repo" doesn't occur with the instructions in this PR.  There was a typo in his check.  So we shall leave the instructions, as in, in this PR for SUSE install.